### ISSUE TITLE
Scenario page: Selected cards seem to have cut corners and incorrect keyboard focus

### ISSUE THREAD

**Issue by romalpani** [dotnet/machinelearning-modelbuilder#1270]
**System Information (please complete the following information):**
 - Model Builder Version: 16.4.1.2112301
 - Visual Studio Version

**Describe the bug**
On the Scenario step, when you select a card, the corners of the selected card seem to be cut. For the same card, try to navigate to it using the keyboard. The keyboard selection (dotted border) is only showing up on the bottom of the card. 

**To Reproduce**

**Expected behavior**
Please use the following Figma mockup for the selected state visual: https://www.figma.com/file/qFoQ4iC27jpNZb7CCuo7Hj/Model-Builder-RM?node-id=977%3A243957

As far as the keyboard focus is concerned, it should be completely visible on the inside of the cards instead of as a border.
Side note: Use the hyperlink color token for the selection state of the card. 

**Screenshots**
![image](https://user-images.githubusercontent.com/48927690/108918058-96a09900-75e5-11eb-82c7-78129907a329.png)

**Additional context**

**Comment by vzhuqin** [comment]
Verified this issue on latest main build: 16.7.0.2137701, now when the keyboard focus is concerned, the dotted border is visible inside the cards border, but the distance between the dotted border and the blue border is a bit to the lower right, could you help to confirm if this is expected behavior?
![image.png](https://images.zenhubusercontent.com/60653e492390a08ea616cb10/dc5def52-e735-4a4e-ac40-1ce44e73526a)

**Comment by vzhuqin** [comment]
@zewditu now the keyboard selection (dotted border) is only showing up on the left and  bottom of the card. could you help to take a look again?
- Microsoft Visual Studio Enterprise 2019: 16.11.2
- ML.Net Model Builder: 16.7.4.2146402 (Main Branch)

![image.png](https://images.zenhubusercontent.com/60653e492390a08ea616cb10/f16973ab-8388-416a-a155-d5e1af28a785)

**Comment by vzhuqin** [comment]
not repro this issue on latest main: 16.9.1.2156904
![image.png](https://images.zenhubusercontent.com/60653e492390a08ea616cb10/e9b88ee0-89db-4605-bd6f-158aeab375ad)
